### PR TITLE
[IOTDB-1992] Add  english user doc for IoTDB REST data service

### PR DESCRIPTION
**Pull request by WilliamSong112**
## Description
Hi all :
I have submitted a pull request for [IOTDB-1992](https://issues.apache.org/jira/browse/IOTDB-1992), please take a look.
This pull request about add  english user doc for IoTDB REST data service.  
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: docs/UserGuide/Communication-Service-Protocol/RestService.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+Parameter Description:
+
+|parameter name  |parameter type |is required|parameter describe|
+|:--- | :--- | :---| :---| 
+|  timestamps | array | yes |  Time column  |
+|  measurements | array | yes  | The name of the measuring point |
+| dataTypes | array | yes  | The data type |
+|  values | array | yes  | Value columns, the values in each column can be `null` |
+|  isAligned | boolean | yes  | Whether to align the timeseries |
+|  deviceId | boolean | yse  | Device name |
```
Comment (SteveYurongSu) [inline docs/UserGuide/Communication-Service-Protocol/RestService.md:167]:
```suggestion
|  deviceId | boolean | yes  | Device name |
```

File: docs/UserGuide/Communication-Service-Protocol/RestService.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+
+
+### Interface
+
+#### ping
+
+Request method:`GET`
+
+Request path:http://ip:port/ping
```
Comment (SteveYurongSu) [inline docs/UserGuide/Communication-Service-Protocol/RestService.md:38]:
```suggestion
Request path: http://ip:port/ping
```

File: docs/UserGuide/Communication-Service-Protocol/RestService.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+### Authentication
+RESTful services use basic authentication. Each URL request needs to carry `'Authorization': 'Basic ' + base64.encode(username + ':' + password)`.
+
+
+
+### Interface
+
+#### ping
+
+Request method:`GET`
```
Comment (SteveYurongSu) [inline docs/UserGuide/Communication-Service-Protocol/RestService.md:36]: A white space is needed after the `:`

Comment (WilliamSong112) [reply docs/UserGuide/Communication-Service-Protocol/RestService.md]: > A white space is needed after the `:`

Hi Yurong , thank's for your review ,I  have fixed these formatting issues  and  update  
 config.js , please take a look.

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**